src/pages/ReportDetailsPage.tsx

```diff
@@ -555,9 +597,11 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                 <ScrollView style={[styles.flex1]}>
                     <View style={[styles.reportDetailsTitleContainer, styles.pb0]}>
                         {renderedAvatar}
-                        {isExpenseReport && nameSectionExpenseIOU}
+                        {isExpenseReport && !shouldShowTitleField && nameSectionExpenseIOU}
                     </View>
 
+                    {isExpenseReport && shouldShowTitleField && nameSectionTitleField}
+
```

Comment:
What's the difference between these two components?

Reply:
`nameSectionExpenseIOU` is non-editable and only for displaying info about IOUs, it's centered and displayed in the same component as avatars.

`nameSectionTitleField` is a MenuItemWithTopDescription which is modifiable. I'll look into unifying this one with `nameSectionGroupWorkspace` since they are fairly similiar.

Reply:
@grgia I've reworked the name sections into `nameSectionStatic` which is based on `DisplayNames` component and is not editable and `nameSectionEditable` which is based on `MenuItemWithTopDescription` and is used for rooms and expense reports when the user has correct permissions.

Reply:
Did you push the `nameSectionStatic` change you're describing?

Reply:
Thank you for the explanation